### DISCUSSION TITLE
Markdown to .enex

### DISCUSSION THREAD

**Discussion by karlbrown-va** [akosbalasko/yarle#404] [Ideas]
Hi, is it possible, or could it be considered, to create a way to generate an .enex from markdown files? Sometimes there are people going the other way - exporting from other notes apps and wanting to import into Evernote, but not as attachments. As far as I can tell the only way to do this is to import a .enex, but I haven't found anything that generates these files from markdown. If such a thing exists please let me know, otherwise is this a feature that might be considered for yarle?

**Comment by akosbalasko** [comment]
Hi @karlbrown-va !

sorry for this long delay to answer. This is an interesting option, i don't know any tools that does this. I always though on Yarle as a one-way converter from enex to md, but it could be considered to extend it to make md-> enex  way possible. Thanks for this good idea!

**Reply by karlbrown-va** [reply]
hi, thanks, I actually ended up creating a tool actually that does this here: https://github.com/karloskalcium/md2enex, but feel free to take ideas or code if you end up putting this feature into Yarle.
